sc2reader/events/game.py

```diff
@@ -399,6 +399,26 @@ def __init__(self, frame, pid, data):
         self.target_data = self.ability_type_data.get("data", None)
 
 
+@loggable
+class CommandManagerStateEvent(GameEvent):
+    """
+    These events indicated that the last :class:`CommandEvent` called has been
+    called again.  For example, if you add three SCVs to an empty queue on a
+    Command Center, the first add will be generate a :class:`BasicCommandEvent`
+    and the two subsequent adds will each generate a
+    :class:`CommandManagerStateEvent`.
+    """
+
+    def __init__(self, frame, pid, data):
+        super(CommandManagerStateEvent, self).__init__(frame, pid)
```

Comment:
This codebase is Python 3 only so…
```suggestion
        super().__init__(frame, pid)
```
https://github.com/asottile/pyupgrade#super-calls

Reply:
Thanks, I was replicating the existing style for consistency, but happy to make this change.

Reply:
Looks like there's still a Python 2 CircleCI check on PRs.